.github/workflows/simulation.yml

```diff
@@ -204,15 +204,15 @@ jobs:
     - uses: actions/checkout@v2
     - name: Bootstrap
       run: |
-        OT_OPTIONS=-DOT_READLINE=OFF sudo apt-get --no-install-recommends install -y expect ninja-build
+        sudo apt-get --no-install-recommends install -y expect ninja-build
     - name: Run
       run: |
-        OT_OPTIONS=-DOT_TIME_SYNC=ON VIRTUAL_TIME=0 ./script/test build expect
+        OT_OPTIONS=-DOT_TIME_SYNC=ON ./script/test build expect
```

Comment:
What determines whether or not we enable virtual time?

Reply:
The script now automatically set VIRTUAL_TIME=0 when running expect tests.

Reply:
Makes sense. We should note this in the script docs:
https://github.com/openthread/openthread/blob/1ad22bac21f01597d86bf6f52aed6fd1a1b00d46/script/test#L270